framework/src/modules/interoperability/stores/chain_account.ts

```diff
@@ -105,6 +105,10 @@ export const allChainAccountsSchema = {
 export class ChainAccountStore extends BaseStore<ChainAccount> {
 	public schema = chainDataSchema;
 
+	public get storePrefix(): Buffer {
```

Comment:
In my understanding this is the override for the base store `storePrefix`. This sounds good, maybe we can consider using a constant for `[0x83, 0xed, 0x0d, 0x25]`